File: modul 4/soal 4/PRAK304-2210817220034-Wanda Karliyanti.c
#include<stdio.h>

main(){
    for(int z=1;z<=5;z++){
int a;
scanf("%d", &a);

if(a==0){
    printf("Nol");}

else if(a>=1 && a<10){
    printf("Satuan");}

else if(a>=11 && a<20){
    printf("Belasan");}

else if(a==10 || a>=20 && a<100){
    printf("Puluhan");}

else{printf("Anda Menginput Melebihi Limit Bilangan");}

}
}

File: modul 4/soal 4/PRAK304-2210817220034-Wanda Karliyanti.py
for z in range(5):
    a=int(input())
    if(a==0):
        print("Nol\n")
    elif(a>=1 and a<10):
        print("Satuan\n")
    elif(a>=11 and a<20):
        print("Belasan\n")
    elif(a== 10 or a>=20 and a<100):
        print("Puluhan\n")
    else:
        print("Anda Menginput Melebihi Limit Bilangan\n")